djautotask/sync_rest.py

```diff
@@ -469,3 +474,92 @@ def sync_related(self, instance):
         )
 
         self.sync_children(*sync_classes)
+
+
+class TaskSynchronizer(SyncRestRecordUDFMixin, Synchronizer):
+    client_class = api.TasksAPIClient
+    model_class = models.TaskTracker
+    udf_class = models.TaskUDF
+    object_filter_field = 'projectID'
+
+    related_meta = {
+        'assignedResourceID': (models.Resource, 'assigned_resource'),
+        'assignedResourceRoleID': (models.Role, 'assigned_resource_role'),
+        'billingCodeID': (models.AllocationCode, 'allocation_code'),
+        'departmentID': (models.Department, 'department'),
+        'phaseID': (models.Phase, 'phase'),
+        'projectID': (models.Project, 'project'),
+        'priority': (models.Priority, 'priority'),
+        'priorityLabel': (models.Priority, 'priority'),
+        'status': (models.Status, 'status'),
+    }
+
+    def __init__(self, *args, **kwargs):
+        self.last_updated_field = 'lastActivityDateTime'
```

Comment:
Quite a bit of this code is the same as in the TicketSynchronizer's `__init__` can you move some of it to a parent class?

Reply:
Thoughts on this as well?

Reply:
Thanks! Those changes look good